cloudera-on-centos/azuredeploy.json

```diff
@@ -1,314 +1,407 @@
-                "description": "Subnet name for the virtual network where resources will be provisioned" 
-            } 
-        },
-        "addressPrefix": { 
-            "type": "string", 
-            "defaultValue": "10.0.0.0/24", 
-            "metadata": { 
-                "description": "The network address space for the virtual network" 
-            } 
-        }, 
-        "subnetPrefix": { 
-            "type": "string", 
-            "defaultValue": "10.0.0.0/24", 
-            "metadata": { 
-                "description": "The network address space for the virtual subnet" 
-            } 
-        }, 
-        "nodeAddressPrefix": { 
-            "type": "string", 
-            "defaultValue": "10.0.0.",
-            "metadata": { 
-                "description": "The IP address prefix that will be used for constructing private IP address for each node in the cluster" 
-            } 
-        }, 
-        "tshirtSize": { 
-            "type": "string", 
-            "defaultValue": "Eval", 
-            "allowedValues": [
-                "Eval",
-                "Prod"
-            ], 
-            "metadata": { 
-                "description": "T-shirt size of the Cloudera cluster (Eval, Prod)" 
-            } 
-        }, 
-        "numberOfDataNodes": {
-            "type": "int",
-            "defaultValue": 3,
-            "metadata": { 
-                "description": "Number of data nodes for Prod, default to 3" 
-            } 
-        },
-        "vmSize": {
-            "type": "string",
-            "defaultValue": "Standard_D14",
-            "metadata": { 
-                "description": "The size of the VMs deployed in the cluster (Defaults to Standard_D14)" 
-            } 
-        },
-        "keyVaultResourceGroup": {
-            "type": "string",
-            "defaultValue": "AzureRM-Util",
-            "metadata": { 
-                "description": "The resource group containing the key vault which provides the private key used for SSH login.  See the readme for details." 
-            } 
-        },
-        "keyVaultName": {
-            "type": "string",
-            "defaultValue": "AzureRM-Keys",
-            "metadata": { 
-                "description": "The name of the key vault which provides the private key used for SSH login.  See the readme for details." 
-            } 
+  "$schema": "http://schema.management.azure.com/schemas/2014-04-01-preview/deploymentTemplate.json",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "adminUserName": {
+      "type": "string",
+      "defaultValue": "testuser",
+      "metadata": {
+        "description": "Admin user name for the Virtual Machines"
+      }
+    },
+    "adminPassword": {
+      "type": "securestring",
+      "defaultValue": "Eur32#1e",
+      "metadata": {
+        "description": "Admin password for the Virtual Machines"
+      }
+    },
+    "cmUsername": {
+      "type": "string",
+      "defaultValue": "cmadmin",
```

Comment:
having default username for linux and cloudera manager should be fine.  Will remove password